framework/src/binary_manager/binary_manager_get_state.c

```diff
@@ -0,0 +1,65 @@
+/****************************************************************************
+ *
+ * Copyright 2020 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied. See the License for the specific
+ * language governing permissions and limitations under the License.
+ *
+ ****************************************************************************/
+
+/***************************************************************************
+ * Included Files
+ ***************************************************************************/
+
+#include <debug.h>
+#include <string.h>
+#include <tinyara/binary_manager.h>
+#include <binary_manager/binary_manager.h>
+#include "binary_manager_internal.h"
+
+int binary_manager_get_state(char *binary_name)
```

Comment:
It is framework, instead of int you should use name of enum which include errors in line 37, 42, 48...